packages/ag-charts-community/src/scene/canvas/hdpiCanvas.ts

```diff
@@ -92,6 +92,10 @@ export class HdpiCanvas {
         Object.freeze(this);
     }
 
+    reset() {
+        this.context.resetTransform();
```

Comment:
We should be doing more than this (e.g. popping all saved state, creating a replacement clean context, or resetting all attributes manually) - but for a patch this is low risk and actually recoverable for the failure case the main fix addresses.